### PR TITLE
FIX: ListItem background color would fail under dark mode

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]

File: screen/settings/settings.js [hunk 1]
```diff
@@ -16,9 +14,6 @@ const styles = StyleSheet.create({
 
 const Settings = () => {
   const { navigate } = useNavigation();
-  // By simply having it here, it'll re-render the UI if language is changed
```
Comment (Overtorment) [inline screen/settings/settings.js:19]: That looks important tho

**Review by Overtorment — APPROVED** [review]

File: screen/settings/settings.js (hunk 1 above)
Comment (marcosrdz) [reply screen/settings/settings.js:19]: No longer needed. RN got its act together.

Comment (marcosrdz) [reply screen/settings/settings.js:19]: oh damn you're right...I misread it for color. language bug is still there. revering file

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ebrmxvaribrtq7vesssdprs56y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/358)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/btr3oum2frwe75vanx27iwjcpm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/361)

**Pull request merged**